config/locales/hi.yml

```diff
@@ -0,0 +1,41 @@
+
+hi:
+  errors:
+    messages:
+      taken_in_past: पहले लिया गया 
+      equal_to_current_password: इससे अलग पासवर्ड होना चाहिए 
+      password_complexity:
+        digit:
+          one: एक अंक होना चाहिए 
+          other: कम से कम %{count} अंक होने चाहिए  
+        lower:
+          one: एक लोअर-केस अक्षर होना चाहिए 
+          other: कम से कम %{count} अक्षर होने चाहिए  
+        symbol:
+          one: एक प्रतीक चिन्ह होना चाहिए 
+          other: कम से कम %{count} प्रतीक चिन्ह होने चाहिए  
+        upper:
+          one: एक अपर-केस अक्षर होना चाहिए 
+          other: कम से कम %{count} अपर-केस अक्षर होने चाहिए  
+  devise:
+    invalid_captcha: कॅप्टचा सही नही था 
+    invalid_security_question: सुरक्षा उत्तर गलत था 
+    paranoid_verify:
+      code_required: सपोर्ट टीम द्वारा दिया गया कोड डाले 
+    paranoid_verification_code:
+      show:
+        submit_verification_code: वेरिफिकेशन कोड डाले 
+        verification_code: वेरिफिकेशन कोड 
+        submit: सबमिट 
+    password_expired:
+      updated: नया पासवर्ड संरक्षित किया 
+      change_required: पासवर्ड अमान्य हो चुका, पासवर्ड बदले 
+      show:
+        renew_your_password: पासवर्ड बदले 
+        current_password: अभी का पासवर्ड 
+        new_password: नया पासवर्ड 
+        new_password_confirmation: नया पासवर्ड सत्यापित 
```

Comment:
<img width="1282" alt="Screen Shot 2020-10-08 at 10 55 34 AM" src="https://user-images.githubusercontent.com/1635820/95475990-cb70c400-0954-11eb-972d-d3834077d5ff.png">

Not sure this is right either. The English version is `Confirm new password`.

Reply:
reviewed

Reply:
This definitely feels like a semantic/ordering thing, but I wanted to confirm.

It looks like you were able to update this to "नया पासवर्ड प्रमाणित" which Google is translating to "New password authenticate." 

My English brain wants this to be ordered the other way, more like "Authenticate new password." Is it safe to assume that this is how it would actually be read in Hindi? 

Also is there an alternative to "authenticate?" I tried the [website you shared](https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=confirm) and searched for confirm. Taking the first option, "पुष्टि करना" it seems to make Google happy, but I don't know if it _actually_ makes sense:

![Screen Shot 2020-10-10 at 7 02 44 AM](https://user-images.githubusercontent.com/691365/95653416-997f6f00-0ac6-11eb-9fba-dd93da12e339.png)